robottelo/cli/fm_health.py

```diff
@@ -0,0 +1,47 @@
+"""
+Usage:
+    foreman-maintain health [OPTIONS] SUBCOMMAND [ARG] ...
+
+Parameters:
+    SUBCOMMAND                    subcommand
+    [ARG] ...                     subcommand arguments
+
+Subcommands:
+    list                          List the checks based on criteria
+    list-tags                     List the tags to use for filtering checks
+    check                         Run the health checks against the system
+
+Options:
+    -h, --help                    print help
+"""
+from robottelo.cli.base import Base
+
+
+class Health(Base):
+    """Manipulates Foreman-maintain's health command"""
+
+    command_base = "health"
+
+    @classmethod
+    def check(cls, options=None):
+        """Build foreman-maintain health check"""
+        cls.command_sub = "check"
+        if options is None:
+            options = {}
```

Comment:
options = options or None
```suggestion
        options = options or {}
```
Non-Blocking comment !

Reply:
You can apply this trick everywehre in your code btw !

Reply:
Updated, thanks